broker_app.py

```diff
@@ -46,9 +48,17 @@
 
 SPREADSHEET_UPLOAD_MESSAGE_ERROR = "We experienced a problem while uploading your spreadsheet"
 
-ingest_api = IngestApi()
-spreadsheet_generator = SpreadsheetGenerator(ingest_api)
-spreadsheet_job_manager = SpreadsheetJobManager(spreadsheet_generator, SPREADSHEET_STORAGE_DIR)
+
+def setup():
+    logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+
+    global ingest_api
```

Comment:
We should not be using globals, but sharing the instance properly between the clients.

Reply:
I am taking care of it on my branch.